CHANGELOG.md

```diff
@@ -19,6 +19,7 @@
   - [Deploy] Kubernetes deployment configuration file - [#29](https://github.com/Accenture/reactive-interaction-gateway/pull/29)
   - [Misc] Smoke tests setup and test cases for API Proxy and Kafka + Phoenix messaging - [#42](https://github.com/Accenture/reactive-interaction-gateway/pull/42)
   - [Outbound] Kafka consumer ready check utility function - [#42](https://github.com/Accenture/reactive-interaction-gateway/pull/42)
+  - [Rig] Possibility to set logging level with env var `LOG_LEVEL` globally for RIG - [#50](https://github.com/Accenture/reactive-interaction-gateway/pull/50)
```

Comment:
- "globally for RIG" could be omitted
- I wanted to discuss this already - wouldn't it make more sense to refer to the issue instead of the PR?
  - there should be a Github issue for each changelog entry
  - the PR should contain closes/fixes <issue>, which will link the PR to the issue
  - you don't have to adapt the changelog entry if the actual PR number is different from what you've expected

Reply:
This was my reasoning in changelog PR:

>Decided to use PR numbers/links in notes. Downside is that you have to either guess (look at most recent PR and increment that by 1) your PR number or do one more commit after PR submit. But I think it's better to be redirected to PR and see code immediately rather than be redirected to issue and go through possibly long page to find out proper PR change.

I don't have problem with either approach.